File: MODUL 2/PRAK205-210817210005-Muhammad-Irgi-Fahrezha.c
#include <stdio.h>
#include <math.h>

int main () {

    int tinggi;
    int sisi_miring;
    int alas;
    int keliling;
    int luas;

    scanf("%d", &tinggi);
    scanf("%d", &sisi_miring);

    alas = sqrt((sisi_miring * sisi_miring ) - (tinggi * tinggi));
    luas = 0.5 * alas * tinggi;
    keliling = tinggi + alas + sisi_miring;

    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2\n", luas);

    return 0;
}

File: MODUL 2/PRAK205-210817210005-Muhammad-Irgi-Fahrezha.py
import math

angka = []
while True:
    line = input()
    if not line:
        break
    angka.extend(line.split())

angka = list(map(float, angka))
tinggi, sisi_miring = angka

alas = math.sqrt((sisi_miring*sisi_miring) - (tinggi*tinggi))
luas = 0.5 * alas * tinggi
keliling = tinggi + alas + sisi_miring

print (f"Alas = {alas:.0f} cm",)
print (f"Tinggi = {tinggi:.0f} cm",)
print (f"Keliling = {keliling:.0f} cm",)
print (f"Luas = {luas:.0f} cm^2",)